low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -150,7 +181,10 @@ def get_realstor_encl_data(fru: str):
         return fru_data
 
 
-# if __name__ == "__main__":
-#     storage = StorageMap()
-#     health_data = storage.get_health_info(rpath="nodes[0]>storage[0]>hw>controllers")
-#     print(health_data)
+if __name__ == "__main__":
+    storage = StorageMap()
+    frus = ["controllers", "psus"]
+    for fru in frus:
+        rpath = f"nodes[0]>storage[0]>hw>{fru}"
+        health_data = storage.get_health_info(rpath=rpath)
+        print(health_data)
```

Comment:
nothing imp, but we can test all the resources by simple rpath of "nodes[0]>storage[0]" instead of iterating over each.

Reply:
This is just for testing, added list so that one can include/exclude frus easily. It will be commented out after the reviews.